CONTRIBUTING.md

```diff
@@ -44,6 +44,7 @@ Current Issue Triagers:
 - [@laosb](https://github.com/laosb)
 - [@abernix](https://github.com/abernix)
 - [@lorensr](https://github.com/lorensr)
+- [@klaussner](https://github.com/klaussner)
```

Comment:
@klaussner Added you here (and below) just to reflect the work you've already been doing. No new expectations.